autogen/oai/gemini.py

```diff
@@ -68,14 +72,33 @@ class GeminiClient:
         "max_output_tokens": "max_output_tokens",
     }
 
+    def initialize_vartexai(self, **params):
```

Comment:
Lets rename this function to "_initialize_vartexai" to tell users this function is private.

Reply:
The function is now private